eng/Versions.props

```diff
@@ -90,14 +91,9 @@
     <!-- System.* packages -->
     <!-- If a System.* package is stuck on version 4.3.x, targets .NET Standard 1.x and hasn't been
     updated in years, you most likely DON'T need it, please exercise caution when adding it to the list. -->
-    <SystemPackageVersionVersion>9.0.0</SystemPackageVersionVersion>
-    <SystemCollectionsImmutableVersion>$(SystemPackageVersionVersion)</SystemCollectionsImmutableVersion>
-    <SystemComponentModelCompositionVersion>$(SystemPackageVersionVersion)</SystemComponentModelCompositionVersion>
-    <SystemCompositionVersion>$(SystemPackageVersionVersion)</SystemCompositionVersion>
-    <SystemDiagnosticsDiagnosticSourceVersion>$(SystemPackageVersionVersion)</SystemDiagnosticsDiagnosticSourceVersion>
+    <SystemComponentModelCompositionVersion>9.0.0</SystemComponentModelCompositionVersion>
+    <SystemCompositionVersion>9.0.0</SystemCompositionVersion>
```

Comment:
we could add these two to Version.Details.xml/.props as well, they likely weren't added there since they're only used in VS-related projects so weren't required for source-build.

Reply:
If I update the darc subscription to include those, that will do it?

Reply:
the subscription itself is fine, we only need to add it to Version.Details.xml/.props. Let me do that.